### PR TITLE
Check for double spaces

### PR DESCRIPTION
**Pull request by Nilstrieb**
### What?

Check for double spaces.

### Why?

Because people keep copy pasting them.

### How?

With a simple `contains`  check.

### Testing?

A unit test.

### CODE REVIEW

**Review by Lion-alt — APPROVED** [review]
Looks good

**Review by adrsch04 — APPROVED** [review]
excellent way to clean up the yaml

**Review by kimberlymo — APPROVED** [review]
Nice idea to fix the typing errors with double spaces :)

**Review by Rwael — APPROVED** [review]
Nice!

File: fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/yaml/Action.java [hunk 1]
```diff
@@ -23,7 +23,11 @@ public interface Action {
 
     record TextAction(String text) implements Action {
         @Override
-        public void simpleVerify() {}
+        public void simpleVerify() {
+            if (this.text().contains("  ")) {
+                throw new FabledTechnicalException("Text contains double spaces. Replace all double spaces with a single space.");
```
Comment (dbabvwid) [inline fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/yaml/Action.java:28]: Pleas  replace throw new Error with an replace function

**Review by dbabvwid — CHANGES_REQUESTED** [review]

File: fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/yaml/Action.java (hunk 1 above)
Comment (Nilstrieb) [reply fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/yaml/Action.java:28]: sounds good, but in a follow up PR, it doesn't have to block this one.

Comment (dbabvwid) [reply fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/yaml/Action.java:28]: Just fix it now

**Review by tamer545 — APPROVED** [review]

**Review by dbabvwid — APPROVED** [review]

**Review by lucasmillanm — APPROVED** [review]

**Pull request merged**